src/HDWallet.h

```diff
@@ -80,11 +80,11 @@ class HDWallet {
     /// Derives the address for a coin.
     std::string deriveAddress(TWCoinType coin) const;
 
-    /// Returns the extended private key.
-    std::string getExtendedPrivateKey(TWPurpose purpose, TWCoinType coin, TWHDVersion version) const;
+    /// Returns the extended private key, derivation path used is "m/purpose'/coin'/account'".
+    std::string getExtendedPrivateKey(TWPurpose purpose, TWCoinType coin, TWHDVersion version, uint32_t account = 0) const;
```

Comment:
```suggestion
    std::string getExtendedPrivateKey(TWPurpose purpose, TWCoinType coin, TWHDVersion version, uint32_t account) const;
```

Reply:
With default value method is called in 70+ cases; I've duplicated the CPP method as well, to get rid of default parameter type.